### ISSUE TITLE
merge request 'changes' tab not diffing deleted files

### ISSUE THREAD

**Issue by brann** [gitlabhq/gitlabhq#8870]
they show up in git diff as deleted, along with all the file contents.

On the changes tab to see deleted files it's possible by clicking "show diff stats" which gives a summary of changed files, new files (green), and delted files (red). But it's disabled by default.

All the attention is placed on the actual diff content. It would be good to have deleted files included there. Their removal is conceputally part of the detailed diff, and the absence could be dangerous if the person doesn't go out of their  way to also look at the stats view.

**Comment by raine** [comment]
I agree that it would be better if deleted files were more prominent in merge requests.

**Comment by maxlazio** [comment]
This is confirmed as a bug in 7.10. We will work on a fix for 7.11

**Comment by brann** [comment]
:+1:

should I close this issue @maxlazio ?

**Comment by maxlazio** [comment]
@brann Not until the issue is fixed :)

**Comment by Razer6** [comment]
Thanks for the issue report. This issue has already been fixed in newer versions of GitLab. Due to the size of this project and our limited resources we are only able to support the latest stable release as outlined in our [contributing guidelines](https://gitlab.com/gitlab-org/gitlab-ce/blob/master/CONTRIBUTING.md#issue-tracker). In order to get this bug fix and enjoy many new features please [upgrade](https://gitlab.com/gitlab-org/gitlab-ce/tree/master/doc/update). If you still experience issues at that time please open a new issue following our issue tracker guidelines found in the [contributing guidelines](https://gitlab.com/gitlab-org/gitlab-ce/blob/master/CONTRIBUTING.md#issue-tracker-guidelines).

**Comment by winterblack** [comment]
I'm still getting this issue on the gitlab website. I don't understand what you mean by upgrade.
